cirq-google/cirq_google/engine/simulated_local_processor.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datetime
+
+from typing import Dict, Iterable, List, Optional, Sequence, TYPE_CHECKING, Union
+
+import cirq
+
+from cirq_google.api import v2
+from cirq_google.engine import calibration
+from cirq_google.engine.abstract_local_processor import AbstractLocalProcessor
+from cirq_google.engine.abstract_local_program import AbstractLocalProgram
+from cirq_google.engine.abstract_program import AbstractProgram
+from cirq_google.engine.local_simulation_type import LocalSimulationType
+from cirq_google.engine.simulated_local_job import SimulatedLocalJob
+from cirq_google.engine.simulated_local_program import SimulatedLocalProgram
+
+import cirq_google.engine.validating_sampler as validating_sampler
+
+if TYPE_CHECKING:
+    from cirq_google.serialization.serializer import Serializer
+
+VALID_LANGUAGES = [
+    'type.googleapis.com/cirq.google.api.v2.Program',
+    'type.googleapis.com/cirq.google.api.v2.FocusedCalibration',
```

Comment:
Since `run_calibration` isn't implemented, should we remove `FocusedCalibration` from this list?